client/src/configs/env.ts

```diff
@@ -38,7 +38,7 @@ export const KEYCLOAK_LOGOUT_URL =
 export const OPENSRP_API_BASE_URL =
   process.env.REACT_APP_OPENSRP_API_BASE_URL ||
   'https://opensrp-stage.smartregister.org/opensrp/rest/';
-export type OPENSRP_API_BASE_URL = typeof OPENSRP_API_BASE_URL;
+export type OPENSRP_APIOPENSRP_API_BASE_URL_BASE_URL = typeof OPENSRP_API_BASE_URL;
```

Comment:
There is a typo here. Are we using the exported type? Others have not been exported

Reply:
yeah seems like there are some that have not been exported.